Balloon/app/memstat.cpp

```diff
@@ -202,8 +205,25 @@ BOOL CMemStat::Update()
         m_Stats[idx].val = statex.ullTotalPhys;
         idx++;
 
+        status = memory->Get(
+                             L"CacheBytes",
+                             0,
+                             &var_val,
+                             NULL,
+                             NULL
+                             );
+
+        if (FAILED(status) || (var_val.vt == VT_NULL)) {
+            PrintMessage("Cannot get CacheBytes");
+            var_val.vt = 0;
```

Comment:
nit: It would be cleaner to call Clear() on the variant.

Assigning -1 to vt is not correct though -- pre-existing issue in several places in this function.

```
    _variant_t var_val(123ULL);
    var_val.vt = -1;
    printf("%llu\n", (ULONGLONG)var_val);

Unhandled exception at 0x7767DBE8 in ConsoleApplication1.exe: Microsoft C++ exception: _com_error at memory location 0x008FF744.
```